pkg/storage/snapshot/restore.go

```diff
@@ -744,13 +783,14 @@ func (t *vmRestoreTarget) createDataVolume(dvt kubevirtv1.DataVolumeTemplateSpec
 		newDataVolume.Annotations = make(map[string]string)
 	}
 	newDataVolume.Annotations[RestoreNameAnnotation] = t.vmRestore.Name
+	newDataVolume.Annotations[v1beta1.AnnPrePopulated] = "true"
```

Comment:
If you're feeling ambitious I think it would be nice to rename this import `cdiv1` instead of v1beta1

Reply:
yeah I started doing that and then I thought about the issues Ill have with all this changing to backport lol
but yeah I didnt like this name either. Ill rename

Reply:
done